.restyled.yaml

```diff
@@ -44,13 +44,11 @@ exclude:
     - ".github/**/*" # https://github.com/restyled-io/restyler/issues/73
     - ".github/*" # https://github.com/restyled-io/restyler/issues/73
     - "third_party/**/*"
-    - "third_party/**"
     - "third_party/*"
-    - "third_party"
```

Comment:
Shouldn't it be third_party/*/repo/** ?

Reply:
good point.  this code predates most of the non-third_party stuff in third_party